src/ui.message.js

```diff
@@ -16,11 +16,8 @@ var _ = Mavo.UI.Message = $.Class({
 			[this.mavo.bar? "after" : "start"]: (this.mavo.bar || this.mavo).element
 		});
 
-		if (o.animationOff) {
-			this.element.style.WebkitTransition = "none";
-			this.element.style.WebkitAnimation = "none";
-			this.element.style.animation = "none";
-			this.element.style.transition = "none";
+		if (o.style) {
+			this.element.style = o.style;
```

Comment:
Please don't do that. It doesn't play nicely with other code, since it overwrites the entire inline style of the element. Better have the style param be a nicely readable object literal and use `$.style()` to apply it.